<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LiveJammin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app.qrc">
    <normaloff>:/images/live-jamming.png</normaloff>:/images/live-jamming.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
	background:url(:/images/background_main.png)
}

QToolBar {
	background-color:#575657;
border:none;
}

QInputDialog {
	background-color:#575657;
}

QInputDialog QLabel {
color: #FFFFFF;
}
QInputDialog QPushButton {
background:url(:/images/btn.png) transparent no-repeat;
border:0px;
width:80px;
height:20px;
}
QInputDialog QPushButton:hover {
background:url(:/images/btn_hover.png) transparent no-repeat;
border:0;
width:70px;
height:25px;
}
QInputDialog QPushButton:pressed {
background:url(:/images/btn_hover.png) transparent no-repeat;
border:0;
width:70px;
height:25px;
}

QMenuBar{
background-color:#575657;
}

QMenuBar::item {
     spacing: 3px; /* spacing between menu bar items */
     padding: 1px 10px;
	background-color:#575657;
 }
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="ChannelList" name="channelList">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">#channelList {
	background: rgb(132, 132, 132);
	color: #FFFFFF;
}

ChannelList::branch:has-children:!has-siblings:closed,
ChannelList::branch:closed:has-children:has-siblings {
         border-image: none;
         image: url(:/images/branch-closed.png);
 }

ChannelList::branch:open:has-children:!has-siblings,
ChannelList::branch:open:has-children:has-siblings  {
         border-image: none;
         image: url(:/images/branch-open.png);
 }</string>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ContiguousSelection</enum>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">Channels</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>19</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionCreate_account"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_LiveJaming"/>
   </widget>
   <widget class="QMenu" name="menuChans">
    <property name="title">
     <string>&amp;Channels</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionCreate_Channel"/>
    <addaction name="actionList_channels"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_room"/>
    <addaction name="actionList_rooms"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Friend"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChans"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/images/chan_ico.png</normaloff>:/images/chan_ico.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionCreate_account">
   <property name="text">
    <string>Create account</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F12</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_LiveJaming">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About LiveJaming</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionAdd_Friend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Friend</string>
   </property>
  </action>
  <action name="actionCreate_room">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create room</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionList_rooms">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>List rooms</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionCreate_Channel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create channel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionList_channels">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>List channels</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
